app/Http/Controllers/Admin/MonsterCrudController.php

```diff
@@ -125,17 +144,20 @@ public function setupListOperation()
                     $query->orWhere('text', 'like', '%'.$searchTerm.'%');
                 },
                 'escaped'       => true,
+                'tab'           => 'Function/Closure',
```

Comment:
I think we can move this to miscellaneous or something similar, no need for a tab for this function/closure thing.

Reply:
Done